src/components/Buttons.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Button } from 'indigo-react';
+
+// TODO: use svg icons
+export const ForwardButton = props => <Button icon="→" {...props} />;
+export const DownloadButton = props => <Button icon="⬇" {...props} />;
```

Comment:
In some cases I imagine we may want to swap out the icons on buttons dynamically (ie show a spinner, then show an arrow). Just displaying a different component seems fine, but is that the most sane way to achieve that?

Reply:
I think I'll be creating another component called `LoadableButton` that supports hot-swapping icons and the background loading behavior from figma. For the static buttons this feels right, though.